wfl/Controllers/MenuSelectionsController.cs

```diff
@@ -37,6 +37,17 @@ public class MenuSelectionsController : ApiController
             return Ok(await dbManager.GetEntityByIdAsync<MenuSelection>(id));
         }
 
+        [Route("api/menu-selection/{id}")]
```

Comment:
We should try to use `RoutePrefix` so that the routes are all similar, unless there is a particular reason why we need to break that pattern.